debian/kafka-mqtt/include/etc/confluent/docker/log4j.properties.template

```diff
@@ -0,0 +1,13 @@
+
+log4j.rootLogger={{ env["KAFKA_MQTT_LOG4J_ROOT_LOGLEVEL"] | default('INFO') }}, stdout
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
```

Comment:
This pr: https://github.com/confluentinc/newwave/pull/14 added the thread id and changed slightly the `log4j.properties`. I suggest we keep them identical for the optimal experience.

Reply:
Thanks, updated `ConversionPattern` according to the latest change.